### PR TITLE
Run rubocop -a on google_json storage

### PR DESCRIPTION
**Pull request by icco**

### CODE REVIEW

**Comment by Everlag** [comment]
This is going to add quite a few conflicts to merging in my storage upgrade, can this be reduced to the rubocop style changes till I PR?

**Comment by icco** [comment]
Totally, and I'll review your PR this weekend.

File: lib/fog/storage/google_json/real.rb [hunk 1]
```diff
@@ -18,11 +18,11 @@ def initialize(options = {})
         end
 
         def signature(params)
-          string_to_sign = <<-DATA
-#{params[:method]}
-#{params[:headers]['Content-MD5']}
-#{params[:headers]['Content-Type']}
-#{params[:headers]['Date']}
+          string_to_sign = <<-DATA.unindent
```
Comment (Temikus) [inline lib/fog/storage/google_json/real.rb]: This seems ok, but we need to make sure this doesn't break the signature, as has happened multiple times before. I did a quick sanity check, but just to check - did you test this in a real scenario?

**Comment by Temikus** [comment]
Sorry for the delay, LGTM overall. One nit in comment.

**Pull request closed without merging**